### PR TITLE
README.md: Add warning for GCC4.9 to GCC5 upgrade

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,6 +359,9 @@ $ cd build
 $ make toolchains
 ```
 
+##### Warning :
```
Comment (jbech-linaro) [inline README.md]: Warning -> Note?

**Comment by jbech-linaro** [comment]
With or without my comment fixed ...
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: README.md (hunk 1 above)
Comment (vchong) [inline README.md]: Changed Warning to Note

**Comment by vchong** [comment]
Tag added. Ok to merge?

**Comment by jbech-linaro** [comment]
Yepps!

**Pull request merged**